.github/workflows/fork_pr_benchmark_closed.yaml

```
@@ -0,0 +1,17 @@
+on:
+  pull_request_target:
+    types: [closed]
+
+jobs:
+  archive_fork_pr_branch:
+    name: Archive closed fork PR branch with Bencher
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

```suggestion
      - name: Checkout repository
        uses: actions/checkout@v4
```
Keeping it this way would maintain consistency with other actions.